cli/pull.go

```diff
@@ -79,11 +80,16 @@ func renderOutput(responseBody io.ReadCloser) {
 		fmt.Fprintf(os.Stderr, "failed to read the closing token: %v", err)
 		return
 	}
+
 }
 
 func display(w io.Writer, statuses []ctrd.ProgressInfo, start time.Time) {
 	var total int64
 	for _, status := range statuses {
+		if len(status.ErrorMessage) > 0 {
```

Comment:
len(status.ErrorMessage) > 0  ===> status.ErrorMessage != "" 

Using len(), We will think it's a slice array and so on.